sympy/functions/combinatorial/factorials.py

```diff
@@ -1194,7 +1227,8 @@ def _eval_expand_func(self, **hints):
         if the summation limit is a nonnegative integer.
         """
         from sympy import Product
-        n, k = self.args
```

Comment:
Is there any way to keep the args attribute? Otherwise this looks as I would expect. Let's see what tests say.

Reply:
unfortunately self.args now has 3 components ☹️

Reply:
Oh yes.